documentation/static/examples/wda/iframe-bg-messaging/tab.html

```diff
@@ -1,7 +1,14 @@
 <!DOCTYPE html>
 <html lang="en">
+    <style>
+        body {
+            padding: 40px;
+            font-family: sans-serif;
+            font-size: 20px;
+        }
+    </style>
 <body>
-    My custom tab
+    Send and receive message between backgroundScript and the current tab
```

Comment:
For consistency's sake, used this form previously:
```suggestion
    Send and receive message between background script and the current tab
```